javascript/big-practice/src/scripts/models/noteModel.js

```diff
@@ -134,13 +134,12 @@ export default class NoteModel {
    *
    * @returns {Object} noteItem
    */
-  async editNote(id, title, description) {
+  async editNote(patternNote) {
     try {
-      const noteItem = this.listNotes.find((note) => note.id === id);
-
-      noteItem.title = title;
-      noteItem.description = description;
-      const note = await putNote(id, noteItem);
+      const noteItem = this.listNotes.find((note) => note.id === patternNote.id);
```

Comment:
You use this way to find item by id at deleteNote() and editNote(), I think you should define a function to find item by id for this case